### PR TITLE
Packaging for release 16.1.0

### PR DESCRIPTION
**Pull request by NabeelAhsen**
# Release 16.1.0

* Use Session Token auth strategy by default for new embedded apps #1111
* Create optional `EnsureAuthenticatedLinks` concern to authenticate deep links using Turbolinks #1118

Before submitting the PR, please consider if any of the following are needed:

- [x] Update `CHANGELOG.md` if the changes would impact users
- [x] Update `README.md`, if appropriate.
- [x] Update any relevant pages in `docs/`, if necessary
- [x] For security fixes, the [Disclosure Policy](https://github.com/Shopify/shopify_app/blob/master/SECURITY.md#disclosure-policy) must be followed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+16.1.0
+------
+* Use Session Token auth strategy by default for new embedded apps #1111
+* Create optional `EnsureAuthenticatedLinks` concern to authenticate deep links using Turbolinks #1118
```
Comment (andyw8) [inline CHANGELOG.md:4]: Let's link to the PRs as per the other entries?

Comment (NabeelAhsen) [reply CHANGELOG.md]: Done!

**Comment by NabeelAhsen** [comment]
CI doesn't seem to happy about this release 😢

**Comment by andyw8** [comment]
You need to run `bundle` and commit the updated `Gemfile.lock`.

**Comment by andyw8** [comment]
(it relates some recent changes when we migrated from Travis to GitHub actions).

**Comment by NabeelAhsen** [comment]
Thanks for the heads up!

**Review by andyw8 — APPROVED** [review]

**Pull request merged**